test/test_skill.py

```
@@ -10,12 +10,12 @@
 from ovos_bus_client import Message
 from ovos_plugin_manager.skills import find_skill_plugins
 from ovos_utils.fakebus import FakeBus
-from skill_laugh import LaughSkill
+from ovos_skill_laugh import LaughSkill
 
 
 @pytest.fixture(scope="session")
 def test_skill(
-    test_skill_id="skill-laugh.openvoiceos", bus=FakeBus()
+    test_skill_id="ovos-skill-laugh.openvoiceos", bus=FakeBus()
```

Ruff

Do not perform function call `FakeBus` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)